iris_grib/_save_rules.py

```diff
@@ -15,6 +15,7 @@
 
 import cf_units
 import gribapi
+from gribapi import GRIB_MISSING_LONG
```

Comment:
I note that is it possible to use `gribapi.grib_set_missing` for this.
Do you think that might be preferable @lbdreyer ?

N.B. I have now read up about this, and not all keys can take a 'missing' value.
Notably, some parameters defined by a code-table, typically a single-byte value, may need to be set to 255 for 'not present'.
This, IIUC is not a true 'missing' value but a table-defined value which happens to _mean_ that this param is unused.  And equally, some params (tables) have no such valid value.
This in in the ecmwf documentation FAQs as ["Why can't I set typeOfFirstFixedSurface to 'missing'?"](https://confluence.ecmwf.int/pages/viewpage.action?pageId=212440369)
and ["How can I find out which keys can be set to 'missing'?"](https://confluence.ecmwf.int/pages/viewpage.action?pageId=235610299)